extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -319,9 +319,9 @@ export const postDeployScriptFriendlyName = localize('postDeployScriptFriendlyNa
 
 // Build
 
-export const NetCoreInstallationConfirmation: string = localize('sqlDatabaseProjects.NetCoreInstallationConfirmation', "The .NET Core SDK cannot be located. Project build will not work. Please install .NET Core SDK version 3.1 or update the .NET Core SDK location in settings if already installed.");
-export function NetCoreSupportedVersionInstallationConfirmation(installedVersion: string) { return localize('sqlDatabaseProjects.NetCoreSupportedVersionInstallationConfirmation', "Currently installed .NET Core SDK version is {0}, which is not supported. Project build will not work. Please install .NET Core SDK version 3.1 or update the .NET Core SDK supported version location in settings if already installed.", installedVersion); }
-export const UpdateNetCoreLocation: string = localize('sqlDatabaseProjects.UpdateNetCoreLocation', "Update Location");
+export const DotnetInstallationConfirmation: string = localize('sqlDatabaseProjects.DotnetInstallationConfirmation', "The .NET SDK cannot be located. Project build will not work. Please install .NET Core SDK version 3.1 or update the .NET SDK location in settings if already installed.");
```

Comment:
This string confuses me since it only mentions .net core 3.1 but not .net 5/6. Is there a way to make this string more inclusive of .net versions?

Reply:
would making it ".NET Core SDK version 3.1 or higher" sound better or should .NET 5 and 6 be explicitly mentioned?

Reply:
I will defer to @dzsquared on this one 😄

Reply:
".NET Core SDK version 3.1 or higher" would be best.  We're going to want to watch out for the 3.1 end of support in Dec 2022 but I'd rather the user get this info in context than have to dig into the documentation for which versions to use.